hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/ReadableFileStoreImplTest.java

```diff
@@ -42,7 +42,7 @@ void getsFileMetadataIfFileExists() {
 
         assertNotNull(fileMeta);
 
-        assertEquals(fileId.fileNum(), fileMeta.fileNumber());
+        assertEquals(fileId.fileNum(), fileMeta.fileId().fileNum());
```

Comment:
I think it is better also to compare file id to file id and not just the file num we are using it as an object the fileid

Reply:
Fixed